app/controllers/secrets_controller.rb

```diff
@@ -110,14 +108,15 @@ def audit_fetch resource, version: nil
   end
 
   def error_info
+    resource_visible = resource_visible?
     return { success: true } unless $ERROR_INFO
 
     # If resource is not visible, the error info will say it cannot be found.
     # That is still what we want to report to the client, but in the log we
     # want more accurate 'Forbidden'.
     {
       success: false,
-      error_message: (resource_visible? ? $ERROR_INFO.message : 'Forbidden')
+      error_message: (resource_visible || resource_visible.nil? ? $ERROR_INFO.message : 'Forbidden')
```

Comment:
I don't understand what `resource_visible` checks that `resource_visible?` doesn't check.
Also, please pay attention to CodeClimate note here

Reply:
i think that you can remove line 111 (`resource_visible = resource_visible?`) and just use here `!resource_visible?` instead of `resource_visible.nil?`